### PR TITLE
React Tutorial: To-do app

### PR DESCRIPTION
**Pull request by DJ-MrJay**
In this exercise, I followed the tutorial: _React Tutorial From Scratch: A Step-by-Step Guide (2021)_, where I developed a Todo app.

### CODE REVIEW

File: src/classBased/components/InputTodo.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+          type="text"
+          className="input-text"
+          placeholder="Add Todo..."
+          value={title}
+          name="title"
+          onChange={this.onChange}
+        />
+        <button type="submit" className="input-submit">
+          Submit
+        </button>
```
Comment (AbigiyaTY) [inline src/classBased/components/InputTodo.js:47]: ### Your [InputTodo.js](https://github.com/DJ-MrJay/react-todo-app/pull/1/files#diff-89941f230a757f229a14e17006ff835825d86cb1f366b2d35eea5e9bf67ea015) File

image

[Optional]

-  When I click the button it doesn't work the add only works by the "enter" key so kindly fix it 😊

**Review by AbigiyaTY — APPROVED** [review]
<h1>Approved 👏</h1>

### **Congratulations!**  🎉 

image

<h2><b>Highlights 🚀 ✔️ ✔️</h2></b>

-  :heavy_check_mark: The repo looks professional.
-  :heavy_check_mark: Git/GitHub flow best practices have been followed
-  :heavy_check_mark: README file descriptive
-  :heavy_check_mark: Commit messages meaningful
-  :heavy_check_mark: Following Linters
# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @AbigiyTY in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/functionBased/components/InputTodo.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        placeholder="Add todo..."
+        value={inputText.title}
+        name="title"
+        onChange={onChange}
+      />
+      <button type="button" className="input-submit">
+        <FaPlusCircle
+          style={{ color: 'darkcyan', fontSize: '20px', marginTop: '2px' }}
+        />
+      </button>
```
Comment (AbigiyaTY) [inline src/functionBased/components/InputTodo.js:46]: ### Your [InputTodo.js](https://github.com/DJ-MrJay/react-todo-app/pull/1/files#diff-89941f230a757f229a14e17006ff835825d86cb1f366b2d35eea5e9bf67ea015) File

image

[Optional]

-  When I click the button it doesn't work the add only works by the "enter" key so kindly fix it  😊

Comment (DJ-MrJay) [reply src/functionBased/components/InputTodo.js:46]: Thank you so much. I will fix it.

**Review by MuhmmdUsama — COMMENTED** [review]
## Hi @DJ-MrJay , 

Your project is complete!  There is nothing else to say
Congratulations! 🎉 

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 
- All To-Do app functionality works fine ✔
- Great app design👌

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------

**Pull request merged**